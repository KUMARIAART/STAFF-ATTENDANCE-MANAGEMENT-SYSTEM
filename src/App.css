b{
    margin-left:2vw;
    font-size:2vw;
}