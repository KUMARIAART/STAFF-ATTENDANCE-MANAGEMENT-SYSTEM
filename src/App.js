import './App.css';
import Staff from './Components/staff/Staff';
import Dashboard from "./dashboard/Dashboard"

function App() {
  return (
    <div className='App'>
    
    </div>
  );
}

export default App;
