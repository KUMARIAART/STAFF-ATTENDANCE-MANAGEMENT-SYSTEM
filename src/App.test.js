import { render, cleanup } from '@testing-library/react';
import React from 'react';
import App from './App';

afterEach(cleanup)
it('should render correctly', () => {
  const { asFragment } = render(<App />);
  expect(asFragment(<App />)).toMatchSnapshot();
});

