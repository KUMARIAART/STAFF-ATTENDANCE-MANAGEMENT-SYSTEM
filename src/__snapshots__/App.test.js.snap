// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <div
        class="row mt-5"
      >
        <div
          class="col-md-4"
          style="margin: auto;"
        >
          <form>
            <div
              class="bg-primary text-light text-center py-3 rounded mb-3 mt-3"
            >
              <h2>
                Mark Attendance
              </h2>
            </div>
            <div
              class="mb-4 row"
            >
              <label
                class="col-sm-2 col-form-label"
              >
                Date:
              </label>
              <div
                class="col-sm-10"
              >
                <select
                  aria-label="Default select example"
                  class="form-select"
                >
                  <option>
                    Select date
                  </option>
                  <option
                    value="2023-03-06"
                  >
                    2023-03-06
                  </option>
                  <option
                    value="2023-03-07"
                  >
                    2023-03-07
                  </option>
                  <option
                    value="2023-03-08"
                  >
                    2023-03-08
                  </option>
                  <option
                    style="background-color: rgb(0, 123, 255); color: white;"
                    value="2023-03-09"
                  >
                    2023-03-09
                  </option>
                  <option
                    value="2023-03-10"
                  >
                    2023-03-10
                  </option>
                  <option
                    value="2023-03-11"
                  >
                    2023-03-11
                  </option>
                  <option
                    value="2023-03-12"
                  >
                    2023-03-12
                  </option>
                  <option
                    value="2023-03-13"
                  >
                    2023-03-13
                  </option>
                  <option
                    value="2023-03-14"
                  >
                    2023-03-14
                  </option>
                  <option
                    value="2023-03-15"
                  >
                    2023-03-15
                  </option>
                  <option
                    value="2023-03-16"
                  >
                    2023-03-16
                  </option>
                </select>
              </div>
            </div>
            <div
              class="mb-4 row"
            >
              <label
                class="col-sm-2 col-form-label"
                for="attendance"
              >
                Mark As:
              </label>
              <div
                class="col-sm-5"
              >
                <div
                  class="form-check form-check-inline"
                >
                  <input
                    class="form-check-input"
                    id="inlineRadio1"
                    name="inlineRadioOptions"
                    style="border: 1px solid grey;"
                    type="radio"
                    value="Present"
                  />
                  <label
                    class="form-check-label"
                    for="inlineRadio1"
                  >
                    Present
                  </label>
                </div>
                <div
                  class="form-check form-check-inline"
                >
                  <input
                    class="form-check-input"
                    id="inlineRadio2"
                    name="inlineRadioOptions"
                    style="border: 1px solid grey;"
                    type="radio"
                    value="Leave"
                  />
                  <label
                    class="form-check-label"
                    for="inlineRadio2"
                  >
                    Leave
                  </label>
                </div>
              </div>
            </div>
            <div
              class="mb-4 row"
            >
              <div
                class="col-sm-12"
              >
                <button
                  class="btn btn-success col-12 bg-primary"
                  disabled=""
                  type="submit"
                >
                  Submit
                </button>
              </div>
            </div>
            <div
              class="mb-3 row"
            >
              <div
                class="col-sm-12 text-center"
              />
            </div>
          </form>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
